src/main/webapp/js/analytics.js

```diff
@@ -16,51 +16,116 @@
 
 /** Sets callbacks for Google Charts instance. */
 google.charts.load('current', {'packages': ['corechart']});
-google.charts.setOnLoadCallback(computeStoreChartsAnalytics);
+google.charts.setOnLoadCallback(computeChartAnalytics);
 
 let storeData;
+let categoryData;
 
-/** Computes and populates DataTable with analytics for stores chart. */
-async function computeStoreChartsAnalytics() {
+/** Computes and populates DataTable with analytics for both charts. */
+async function computeChartAnalytics() {
   const response = await fetch('/compute-analytics');
   const analytics = await response.json();
 
+  console.log(analytics);
+
+  handleStoreChart(analytics);
+  handleCategoryChart(analytics);
+}
+
+/**
+ * Populates DataTable with store data and passes that to draw method.
+ * @param {Array} analytics First entry is store data, second is category.
```

Comment:
This is not an array right? I believe it's json. Comment will probably also need to be changed to reflect that.